src/features/reviews/components/TableView.vue

```diff
@@ -37,7 +37,7 @@
       <tr
         v-for="row in reviewTable.getRowModel().rows"
         :key="row.id"
-        class="h-20 bg-lowBackground"
+        class="group h-20 bg-lowBackground"
```

Comment:
Whats this class doing?

Reply:
enables the hover movie tooltip visibility